packages/smart-wallet/src/smartWallet.js

```diff
@@ -446,10 +469,26 @@ export const prepareSmartWallet = (baggage, shared) => {
             },
             onStatusChange: offerStatus => {
               logger.info('offerStatus', offerStatus);
+
               updatePublishKit.publisher.publish({
                 updated: 'offerStatus',
                 status: offerStatus,
               });
+
+              const isOfferPossiblyExitable = !(
+                'numWantsSatisfied' in offerStatus
+              );
+              if (isOfferPossiblyExitable) {
+                return;
+              }
+
+              state.activeOfferSeats.has(offerStatus.id) &&
+                state.activeOfferSeats.delete(offerStatus.id);
```

Comment:
this isn't frontend :-P
```suggestion
              if (state.activeOfferSeats.has(offerStatus.id)) {
                state.activeOfferSeats.delete(offerStatus.id);
              }
```

Reply:
Should we use https://eslint.org/docs/latest/rules/no-unused-expressions then?

Reply:
Good idea. Its defaults ban more than we'd want but the settings look like it could be narrowed appropriately.

Do you want to PR that or should I?

Also, thanks for looking for a way to automate the review comment. Always better to let the tools do it.